app/services/doc_auth/lexis_nexis/responses/true_id_response.rb

```diff
@@ -178,6 +179,17 @@ def true_id_product
           products[:TrueID] if products.present?
         end
 
+        def convert_id_type(state_id_type)
+          case state_id_type
+          when 'Identification Card'
+            'state_id_card'
+          when 'Permit'
+            'drivers_permit'
+          when 'Drivers License'
+            'drivers_license'
+          end
```

Comment:
we expect the trueid/acuant responses to stay in sync with each other, right? WDYT about making a shared hash we look these up in?

```
ID_TYPE_SLUGS = {
  'Identification Card' => 'state_id_card',
  'Permit' => 'drivers_permit',
  'Drivers License' => 'drivers_license',
}
# ...
pii[:state_id_type] = ID_TYPE_SLUGS[pii[:state_id_type]]
```